templates/dynamic-networks-controller.yaml.j2

```diff
@@ -117,8 +120,8 @@ spec:
               readOnly: true
             - name: multus-server-socket
               mountPath: /host/run/multus/multus.sock
-            - name: containerd-socket
-              mountPath: {{ CRI_SOCKET_PATH }}
+            - name: cri-socket
+              mountPath: /host{{ CRI_SOCKET_PATH }}
```

Comment:
You can remove the attribute from here. You can parametrize the host path in `volumes` to fit the system. Inside the container you may be able to keep it always on the same path, removing some of the complexity

Reply:
In the host, I'll have either:
- /run/containerd/containerd.sock
- /run/crio/crio.sock

IIUC, your suggestion is to mount to the pod something like: `/host/cri-socket.sock` . 

Not saying this is bad, but I don't see the proposed alternative as more complex than hard-coding.

Plus, if I have access to the container - but not its configuration - I will **always** know which type of runtime I'm using.
(I do wonder if this scenario is common ...)

Reply:
I prefer keeping an abstract path as an "interface" in a way, so the manifest serves as a mapper between the two realms. I don't have any stronger reasoning for it, so I won't fight